tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -714,6 +714,26 @@ public function userEnablesSyncOfShareUsingTheGraphApi(string $user, string $sha
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When user :user tries to enable share sync of non-existing resource using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userTriesToEnablsShareSyncOfNonExistingResourceUsingTheGraphApi(string $user):void {
+		$shareSpaceId = FeatureContext::SHARES_SPACE_ID;
+		$response =  GraphHelper::enableShareSync(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$this->featureContext->getActualUsername($user),
+			$this->featureContext->getPasswordForUser($user),
+			WebDavHelper::generateUUIDv4(),
```

Comment:
Does that matter if we send the itemId in `WebDavHelper::generateUUIDv4()` or in the complete format of the itemId? If not then others LGTM.